nextflow_schema.json

```diff
@@ -48,7 +48,7 @@
                     "type": "string",
                     "description": "The type of transformation to perform.",
                     "default": "lock",
-                    "enum": ["lock", "age"]
+                    "enum": ["lock", "age", "earliest"]
```

Comment:
I think there should be a parameter here and in nextflow.config to adjust the earliest header?

Reply:
I thought so too, so I added the option in `transform.py`, but the user story for this states:

"Users cannot change the name of "earliest_date" column"

So I made it unchangeable.

Reply:
@emarinier Thanks for pointing that out. Yes, I think it's best to include as a parameter. Could even mark as hidden by default in the pipeline code itself (can always override to unhide if needed).

That way, if we ever need to change the column name for some reason, we can do it without updating the pipeline.

Reply:
Updated here: ee079f7